admin/src/css/theme.less

```diff
@@ -145,3 +145,7 @@ ul {
 .btn-bottom-spacing  {
   margin-bottom: 5px;
 }
+.no-padding-horizontal {
+   padding-left: 0 !important;
+   padding-right: 0 !important;
```

Comment:
You should try to never use `!important` - it makes life really hard later because you can't overwrite it. Instead come up with a CSS selector that has higher specificity than the one you're overwriting.